gazebo_plugins/CMakeLists.txt

```diff
@@ -198,6 +198,7 @@ ament_export_libraries(gazebo_ros_tricycle_drive)
 add_library(gazebo_ros_video SHARED
   src/gazebo_ros_video.cpp
 )
+target_include_directories(gazebo_ros_video PUBLIC include)
```

Comment:
`gazebo_ros_video` is not being able to find `image_transport` includes on CI:

```
19:44:23 [79.272s] [K/home/jenkins-agent/workspace/ci_linux/ws/src/Karsten1987/gazebo_ros_pkgs/gazebo_plugins/src/gazebo_ros_video.cpp:26:[Kfatal erro[Kimage_transport/image_transport.h: No such file or directory
19:44:23 [79.272s]  #include [K<image_transport/image_transport[K
19:44:23 [79.272s]           [K^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~[K
19:44:23 [79.272s] compilation terminated.
```

Reply:
@Karsten1987, did you have a chance to look into this?

Reply:
I did not, but the fix should be trivial; Essentially adding "image_transport" to the list of `ament_target_dependencies`